NeuroID/src/debug/java/com/neuroid/tracker/storage/NIDDataStoreManager.kt

```diff
@@ -50,7 +50,7 @@ private object NIDDataStoreManagerImp: NIDDataStoreManager {
 
     @Synchronized
     override fun saveEvent(event: NIDEventModel) {
-        if (NIDJobServiceManager.isStopped() && event.type != CLOSE_SESSION) {
+        if (NIDJobServiceManager.isStopped()) {
```

Comment:
Didn't we just put in this close session check for the force send?

Reply:
As discussed, updates to NeuroID.start and NeuroID.stop ensure that the CLOSE_SESSION event is created and sent while the job service manager is running.